commands/create-account.js

```diff
@@ -48,10 +48,23 @@ async function createAccount(options) {
         keyPair = await KeyPair.fromRandom('ed25519');
         publicKey = keyPair.getPublicKey();
     }
-    await near.createAccount(options.accountId, publicKey);
     if (keyPair) {
         await near.connection.signer.keyStore.setKey(options.networkId, options.accountId, keyPair);
     }
+    try {
+        await near.createAccount(options.accountId, publicKey);
+    } catch(error) {
+        if (error.message.includes('Timeout')) {
```

Comment:
I think we already should have `type` field on `error` to avoid searching  string in message.

Reply:
I don't understand why, but I tried using `await` and it didn't work.

Reply:
I wish, the `type` is `UntypedError` which I really don't understand. In the PR description I included a patch to hack nearcore to always return Server Timeout and you can see this behavior.

Reply:
I think included nearcore patch is not returning error in same way as it is normally returned. 

Normally it is:

```rust
.map_err(|err| RpcError::server_error(Some(ServerError::from(err))))
```
i.e. it is converted/wrapped into `RpcError::server_error` and otherwise returned as `Ok` response

in your patch it is:

```rust
Err(timeout_err())
```

- using `timeout_err` directly without conversion/wrapping
- using `Err` instead of `Ok`

Reply:
I just had a couple timeouts trying to create a new account (now those 2 accounts are bricked, no way to recover the key)
The errors are the same.

<img width="2472" alt="Screenshot 2020-06-09 15 02 20" src="https://user-images.githubusercontent.com/1042667/84205709-88fcb080-aa62-11ea-91b3-5a4b50612f36.png">

Reply:
@bowenwang1996 @fckt do you have any idea what is going on here? Shouldn't nearcore already return typed errors in every case?

Reply:
I am very sorry but looks like it still hasn't been fixed. We scammed you :(

Reply:
@mikedotexe looks like we have to roll with `error.message.includes` for a while then